### PR TITLE
Better message

### PR DESCRIPTION
**Pull request by deltakosh**

### CODE REVIEW

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

File: packages/tools/nodeEditor/src/graphSystem/connectionPointPortData.ts [hunk 1]
```diff
@@ -163,8 +163,14 @@ export class ConnectionPointPortData implements IPortData {
         }
 
         switch (issue) {
-            case NodeMaterialConnectionPointCompatibilityStates.TypeIncompatible:
-                return `Cannot connect two different connection types:\nSource is ${NodeMaterialBlockConnectionPointTypes[this.data.type]} and destination is ${NodeMaterialBlockConnectionPointTypes[(targetPort.data as NodeMaterialConnectionPoint).type]}`;
+            case NodeMaterialConnectionPointCompatibilityStates.TypeIncompatible: {
+                const port = targetPort.data as NodeMaterialConnectionPoint;
+                let acceptedTypes = port.acceptedConnectionPointTypes.map((t) => NodeMaterialBlockConnectionPointTypes[t]).join(", ");
+
+                acceptedTypes = `${NodeMaterialBlockConnectionPointTypes[port.type]}` + (acceptedTypes ? `,${acceptedTypes}` : "");
```
Comment (Popov72) [inline packages/tools/nodeEditor/src/graphSystem/connectionPointPortData.ts:170]: nit:

```suggestion
                acceptedTypes = `${NodeMaterialBlockConnectionPointTypes[port.type]}` + (acceptedTypes ? `, ${acceptedTypes}` : "");
```

Same below.

**Review by sebavan — COMMENTED** [review]
LGTM, letting @Popov72 the final approval to auto merge :-)

**Review by Popov72 — APPROVED** [review]

**Comment by bjsplat** [comment]
Snapshot stored with reference name:
refs/pull/15819/merge

Test environment: 
https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15819/merge/index.html

To test a playground add it to the URL, for example:

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15819/merge/index.html#WGZLGJ#4600

Links to test babylon tools with this snapshot:

https://playground.babylonjs.com/?snapshot=refs/pull/15819/merge
https://sandbox.babylonjs.com/?snapshot=refs/pull/15819/merge
https://gui.babylonjs.com/?snapshot=refs/pull/15819/merge
https://nme.babylonjs.com/?snapshot=refs/pull/15819/merge

To test the snapshot in the playground with a playground ID add it after the snapshot query string:

https://playground.babylonjs.com/?snapshot=refs/pull/15819/merge#BCU1XR#0

**Comment by bjsplat** [comment]
WebGL2 visualization test reporter:

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15819/merge/testResults/webgl2playwright/index.html

**Pull request merged**